mud-contracts/world/src/modules/kill-mail/systems/KillMail.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.21;
+
+import { EveSystem } from "@eveworld/smart-object-framework/src/systems/internal/EveSystem.sol";
+import { ResourceId } from "@latticexyz/store/src/ResourceId.sol";
+import { KillMailLoss } from "../../../codegen/common.sol";
+import { KillMailTable } from "../../../codegen/tables/KillMailTable.sol";
+import { IKillMailErrors } from "../IKillMailErrors.sol";
+import { Utils } from "../Utils.sol";
+
+contract KillMail is EveSystem {
+  function reportKill(
+    uint256 killMailId,
+    address killer,
+    address victim,
+    KillMailLoss loss,
+    uint256 timestamp
```

Comment:
is this is the timestamp the kill was reported or the time its stored on-chain ?

Reply:
The in-game timestamp for the kill.

Reply:
Changed to `killTimestamp`.